modules/virt-understanding-node-maintenance.adoc

```diff
@@ -1,25 +1,24 @@
 // Module included in the following assemblies:
 //
-// * virt/node_maintenance/virt-node-maintenance.adoc
-// * virt/node_maintenance/virt-setting-node-maintenance.adoc
+// virt/node_maintenance/virt-about-node-maintenance.adoc
 
-[id="virt-understanding-node-maintenance{context}"]
+[id="virt-understanding-node-maintenance_{context}"]
 = Understanding node maintenance mode
 
-Placing a node into maintenance marks the node as unschedulable and drains all 
-the virtual machines and pods from it. Virtual machine instances that have a 
-`LiveMigrate` eviction strategy are live migrated to another node without loss 
-of service. This eviction strategy is configured by default in virtual machine 
-created from common templates but must be configured manually for custom 
-virtual machines. 
+Nodes can be placed into maintenance mode using the `oc adm` utility, or using `NodeMaintenance` custom resources (CRs).
 
-Virtual machine instances without an eviction strategy will be deleted on the 
-node and recreated on another node. 
+Placing a node into maintenance marks the node as unschedulable and drains all the virtual machines and pods from it. Virtual machine instances that have a `LiveMigrate` eviction strategy are live migrated to another node without loss of service. This eviction strategy is configured by default in virtual machine created from common templates but must be configured manually for custom virtual machines.
+
+Virtual machine instances without an eviction strategy will be deleted on the node and recreated on another node.
```

Comment:
This statement is incorrect. Something more along the lines of:

Virtual Machine Instances without an eviction strategy will be shut down. VM's with a RunStrategy of `Running` or `RerunOnFailure` will be recreated on another node. See (link to RunStrategy Documentation) for more information.

Reply:
The distinction here is that VMIs that aren't backed by a VM or VM's that have a RunStrategy of `Manual` will not be restarted. This is expected behavior.

Reply:
Thanks @stu-gott updated with your change: https://deploy-preview-29501--osdocs.netlify.app/openshift-enterprise/latest/virt/node_maintenance/virt-about-node-maintenance.html

We can't do inline links in the OCP docs build, so I've added a link to RunStrategy to the related links section